apps/admin-gui/src/app/shared/components/role-filter-inputs/group-roles-filter.component.html

```diff
@@ -0,0 +1,29 @@
+<mat-form-field class="me-2 fix-select">
+  <mat-label>{{'SHARED.COMPONENTS.GROUP_ROLES_FILTER.FILTER_ROLES' | translate}}</mat-label>
+  <mat-select (selectionChange)="changeRoles()" [formControl]="roles" multiple>
+    <mat-select-trigger>
+      {{displaySelectedRoles() | groupRole}}
+    </mat-select-trigger>
+    <mat-option
+      class="vo-theme"
```

Comment:
Don't think it should be static vo-theme, since it is used on pages of other objects.
```suggestion
      class="{{theme}}"
```